src/placesAutocompleteNew.js

```diff
@@ -71,6 +71,7 @@ function placesAutocompleteNew({ googleApiKey, searchLength, languageCode = 'en'
           languageCode: languageCode,
           input: actualQuery,
           sessionToken,
+          ...(filters && { "includedPrimaryTypes": filters }),
```

Comment:
filters sounds a bit too vague, I would call it `includedPrimaryTypes` or `includeTypes`

```suggestion
          ...(includeTypes && { "includedPrimaryTypes": includeTypes }),
```